onnxruntime/core/common/cpuid_info_vendor.cc

```diff
@@ -232,11 +232,15 @@ void CPUIDInfo::VendorInfoInit() {
   }();
 
   const auto* vendor_info = FindCpuVendorInfo(vendor);
+// Do below logging only if CPUINFO_SUPPORTED by the OS and still vendor_info is not available.
```

Comment:
I think that it is an implementation detail that the vendor info lookup depends on cpuinfo. I also think that if the lookup is unsuccessful, it is worth logging.

do you think that this information should be logged at a different level?